src/config-file.c

```diff
@@ -316,6 +319,12 @@ Config* load_config_file(const gchar *config_file, GError **error)
                 return NULL;
         }
 
+        if (!bundle_location_given && !config->stream_bundle) {
+                g_set_error(error, G_KEY_FILE_ERROR, G_KEY_FILE_ERROR_KEY_NOT_FOUND,
+                            "bundle_download_location is required if stream_bundle is disabled");
```

Comment:
To make it more obvious, probably

```suggestion
                            "'bundle_download_location' is required if 'stream_bundle' is disabled");
```

Reply:
Done.